### PR TITLE
Add AWS url

### PR DESCRIPTION
**Pull request by VladimirMikhaylov**

### CODE REVIEW

File: src/faucet/Faucet.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import axios from 'axios';
+import ServiceFactory from '../services/ServiceFactory';
+import transactionMapper from '../addresses/TransactionMapper';
+import FaucetBlock from './LeftBlock';
+
+// import { broadcast, sponsorship, transfer, waitForTx } from '@acryl/acryl-transactions'
```
Comment (markuzti) [inline src/faucet/Faucet.js]: delete debugging info

File: src/faucet/Faucet.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+// const ttx = transfer({ recipient: '3JHj9mVz1URexGe4MHCCEMBm4gJTkhqJnv5', amount: 270000000, feeAssetId: null }, MASTER_SEED)
+// broadcast(ttx, API_BASE)
+import TransactionList from '../addresses/TransactionList';
+
+export default class Faucet extends React.Component {
+    state = {
+        transactions: [],
+        address: '3JNbXos1asm3ZG47QG1MWjknQhvMDqrrH2v'  
+    }
+    setAcryl(str) {
```
Comment (markuzti) [inline src/faucet/Faucet.js]: whats it?

File: src/faucet/Faucet.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        return str.replace('WAVES', 'ACRYL');
+    }
+
+    componentDidMount(){
+        const addressService = ServiceFactory.addressService();
+        const address = this.state.address;
+        addressService.loadTransactions(address).then(transactions => {
```
Comment (markuzti) [inline src/faucet/Faucet.js]: await addressService.loadTransactions(address)

File: src/faucet/LeftBlock.js [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+        this.addressChange = this.addressChange.bind(this);
+        this.getCoins = this.getCoins.bind(this);
+    }
+    onChange =  async(value)=> {
+        if(value == null){
+            await this.setState({isSubmiting: false});
+        }else{
+            await this.setState({isSubmiting: true, token: value});
+        }
+        await console.log("Captcha value:", value);
```
Comment (markuzti) [inline src/faucet/LeftBlock.js]: without 'await'

Comment (markuzti) [reply src/faucet/LeftBlock.js]: console.log needed here?

File: src/faucet/LeftBlock.js [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+            address: '', 
+            isSubmiting:false,
+            token:'',
+            responseMessage:'',
+        };
+
+        this.addressChange = this.addressChange.bind(this);
+        this.getCoins = this.getCoins.bind(this);
+    }
+    onChange =  async(value)=> {
```
Comment (markuzti) [inline src/faucet/LeftBlock.js]: why do u use here function expression? plz, use function declaration

File: src/faucet/LeftBlock.js [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+    }
+ 
+    addressChange(event) {
+        this.setState({address: event.target.value});
+    }
+
+    getCoins(event) {
+        // console.log('Отправленное имя: ' + this.state.address);
+        event.preventDefault();
+        axios.post('https://3h765xjw97.execute-api.eu-central-1.amazonaws.com/faucet', {
```
Comment (markuzti) [inline src/faucet/LeftBlock.js]: move URL to config

File: src/faucet/LeftBlock.js [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+        const recaptchaValue = recaptchaRef.current.getValue();
+        this.props.onSubmit(recaptchaValue);
+    }
+ 
+    addressChange(event) {
+        this.setState({address: event.target.value});
+    }
+
+    getCoins(event) {
+        // console.log('Отправленное имя: ' + this.state.address);
```
Comment (markuzti) [inline src/faucet/LeftBlock.js]: delete this string

File: src/faucet/LeftBlock.js [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+                        <p style={{fontWeight:"bold"}}>Testnet address</p>
+                        Fill out your Testnet address to receive 10 ACRYL
+                    </label>
+                    <input type="text" value={this.state.address} onChange={this.addressChange} className='AdressInput'/>
+                    <label>
+                        Please enter the Captcha
+                    </label>
+                    <ReCAPTCHA
+                        ref={this.recaptchaRef}
+                        sitekey="6LcOH9gUAAAAAGTtomA98d1nW-JmLUC63E3XX6gl"
```
Comment (markuzti) [inline src/faucet/LeftBlock.js]: Move key to config

File: src/faucet/LeftBlock.js [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+                        onChange={this.onChange}
+                    />
+                    
+                    <input type="submit" value="Отправить" disabled={!this.state.isSubmiting} className='SubbmitButton'/>
```
Comment (markuzti) [inline src/faucet/LeftBlock.js]: dont use ru locale

File: src/faucet/Faucet.js (hunk 2 above)
Comment (deadblackclover) [reply src/faucet/Faucet.js]: setAcryl use for replace WAVES name asset on ACRYL

File: package.json [hunk 10]
```diff
@@ -14,6 +14,7 @@
     "app:prod": "webpack --mode production --config=webpack.config.js"
   },
   "dependencies": {
+    "@acryl/acryl-transactions": "^3.16.7",
```
Comment (deadblackclover) [inline package.json]: Delete not use package

File: src/faucet/Faucet.js [hunk 11]
```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import axios from 'axios';
+import ServiceFactory from '../services/ServiceFactory';
+import transactionMapper from '../addresses/TransactionMapper';
+import FaucetBlock from './LeftBlock';
+
+// import { broadcast, sponsorship, transfer, waitForTx } from '@acryl/acryl-transactions'
+
+// export const MASTER_SEED = 'test acc 2'
```
Comment (deadblackclover) [inline src/faucet/Faucet.js]: Мade public seed

File: src/App.js [hunk 12]
```diff
@@ -47,10 +48,11 @@ class App extends React.Component {
                             <Route exact path={routes.blocks.one(routeParams.blockHeight)} component={SingleBlockPage} />
                             <Route path={routes.nodes.list} component={NodesPage} />
                             <Route path={routes.peers.list} component={PeersPage} />
+                            <Route path={routes.faucet.list} component={Faucet} />
                             <Route exact path={routes.transactions.one(routeParams.transactionId)} component={SingleTransactionPage} />
                             <Route exact path={routes.addresses.one(routeParams.address)} component={SingleAddressPage} />
                             <Route exact path={routes.aliases.one(routeParams.alias)} component={SingleAliasPage} />
-                            <Route path={routes.root} component={MainPage} />
+                            <Route path={routes.root} component={MainPage} />
```
Comment (deadblackclover) [inline src/App.js]: nit: odd space

**Pull request merged**